src/util/protection.cpp

```diff
@@ -85,4 +85,4 @@ namespace big::protection
 				return true;
 		return false;
 	}
-}
```

Comment:
Pointless whitespace change.

Reply:
It might be a result of difference in IDE setup, some IDE's add a newline at the end some remove it.